File: 2.js
var fib1 = 0;
var fib2 = 1;
var sum = 0;

for (var i = fib1 + fib2; i <= 4000000; i = fib1 + fib2) {
  if (i % 2 === 0) {
    sum += i;
  }
  fib1 = fib2;
  fib2 = i;
};

alert(sum);

File: rb/2.rb
fib1 = 0
fib2 = 1
sum = 0
i = 0
printer = 0

while sum < 4000000
  i = fib1 + fib2
  fib1 = fib2
  fib2 = i
  sum = i
  if (sum % 2 == 0)
    printer += sum
  end
end
puts printer
